include/unifex/dematerialize.hpp

```diff
@@ -198,7 +181,7 @@ namespace _demat {
 namespace _demat_cpo {
   inline constexpr struct _fn {
    private:
-    template<bool>
+    template <bool>
```

Comment:
Is the space here significant?

Reply:
I changed it to be consistent with the formatting elsewhere. For the purposes of reviewing the PR, you can change the settings to hide whitespace changes.